src/main/groovy/com/github/jrubygradle/internal/GemVersionResolver.groovy

```diff
@@ -4,10 +4,52 @@ import org.gradle.api.artifacts.DependencyResolveDetails
 
 class GemVersionResolver {
 
-    private Map versions = [:]
-  
+    Map versions
+    private final Object configuration
+    private final Object logger
+
+    static void setup(Object project) {
+        project.configurations.each {
```

Comment:
Won't applying this to each configuration change the behavior of java dependency resolution in other configurations, such as `compile` or `test` (in the case of a hybrid project)?